dockerfile/README.md

```diff
@@ -1,54 +1,64 @@
+    + `https:<DOCKER_HOST>:9443/carbon`
+    
+>In here, <DOCKER_HOST> refers to hostname or IP of the host machine on top of which containers are spawned.
 
- Follow the steps to build the WSO2 Data Analytics Server 3.1.0 Docker image and run in your local machine.
- 
- The local copy of the `Dockerfile` directory will be referred as, `DOCKERFILE_HOME`.
- 
- * Add the JDK and WSO2 Data Analytics Server distributions:
-     - Download JDK 1.8 (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and copy it to `<DOCKERFILE_HOME>/files`.
-     - Download the WSO2 Data Analytics Server 3.1.0 distribution (https://wso2.com/analytics) and copy it to `<DOCKERFILE_HOME>/files`.
-   Please refer [WSO2 Update Manager documentation](https://docs.wso2.com/display/ADMIN44x/Updating+WSO2+Products) to obtained the WSO2 Data Analytics Server 3.1.0
-   with latest bug fixes and updates.
- 
- * Build the Docker image:
-     - Navigate to `<DOCKERFILE_HOME>` directory.
-     - Execute the `docker build` command as shown below:
-         + `docker build -t wso2das:3.1.0 .`
- 
- * Docker run:
-     - Run the WSO2 Data Analytics Server 3.1.0 Docker container as follows;
-         + `docker run -it -p 9443:9443 wso2das:3.1.0`
-         
-       **Note**: Here, only port 9443 (HTTPS servlet transport) has been mapped to a Docker host port.
-       You may map other container service ports, which have been exposed to Docker host ports, as desired.
- 
- * Access management console:
-     -  To access the management console, use the Docker host IP and port 9443.
-         + `https://<DOCKER_HOST>:9443/carbon`
 
 ## How to update configurations
+Configurations would lie on the Docker host machine and they can be volume mounted to the container. <br>
+As an example, steps required to change the port offset using `carbon.xml` is as follows.
+
+##### 1. Stop the Data Analytics Server container if it's already running.
+In WSO2 Data Analytics Server 3.1.0 product distribution, `carbon.xml` configuration file <br>
+can be found at `<DISTRIBUTION_HOME>/repository/conf`. Copy the file to some suitable location of the host machine, <br>
+referred to as `<SOURCE_CONFIGS>/carbon.xml` and change the offset value under ports to 1.
 
-The configurations will be maintained on the Docker Host machine and volume mounted to the container.
+##### 2. Grant read permission to `other` users for `<SOURCE_CONFIGS>/carbon.xml`
+```
+chmod o+r <SOURCE_CONFIGS>/carbon.xml
+```
 
-As an example, the steps required to change the port offset in `carbon.xml` is detailed.
+##### 3. Run the image by mounting the file to container as follows.
+```
+docker run 
+-p 9444:9444
+--volume <SOURCE_CONFIGS>/carbon.xml:<TARGET_CONFIGS>/carbon.xml
+wso2das:3.1.0
+```
 
-* Stop the Data Analytics Server container if it's already running.
+>In here, <TARGET_CONFIGS> refers to /home/wso2carbon/wso2das-3.1.0/conf folder of the container.
```

Comment:
/home/wso2carbon/wso2das-3.1.0/conf should be changed to /home/wso2carbon/wso2das-3.1.0/repository/conf.